packages/manager/modules/overthebox/src/overthebox/migration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "overthebox_migration_title": "Changement d'offre",
+  "overthebox_migration_offers_error": "Une erreur s'est produite lors de la récupération des offres disponibles. {{error}}",
```

Comment:
I don't know the convention, but when there is an error I put : instead of .

Reply:
I don't understand what you want me to do with "instead of".